submissions/description/broken_access_control/username_enumeration/non_brute_force/template.md

```diff
@@ -0,0 +1,21 @@
+# Username Enumeration (Non-Brute Force)
+
+## Overview of the Vulnerability
+
+Username enumeration is a vulnerability where an attacker is able to confirm or guess correct usernames through a difference in the server’s response to input. It often occurs on login, registration, and password reset pages. This application has a manual username enumeration vulnerability which allows an attacker to manually guess email and password combinations.
```

Comment:
The terms used in this are not accurate, as a non-bruteforce method allows an attacker to view all usernames without submitting it "manually". I've updated the terminology below.

> Username enumeration is a vulnerability where an attacker is able to confirm or guess correct usernames through a difference in the server’s response to input. It often occurs on login, registration, and password reset pages. This application has a username enumeration vulnerability which allows an attacker to identify the username or email of a user without brute forcing it, allowing an attacker to gain this user information for all users within the application in a short period of time.